### ISSUE TITLE
Copyright metadata in RDFa

### ISSUE THREAD

**Issue by sujato** [suttacentral/legacy-suttacentral#194]
### TL;DR

Copyright data should be clearly marked up and machine-readable according to latest best practices of W3C/RDFa, so far as is reasonable given the unclear nature of many of our licences. 

### Discussion

So far we include copyright info in our files, displayed in the sidebar, but not in the actual page metadata. Compare, say Wikipedia, which in the head has:

    <link rel="copyright" href="//creativecommons.org/licenses/by-sa/3.0/"/>

I would like to propose that we do the same. Not necessarily the same format, we should research best practices. Presumably we should follow RDFa. The main purpose of this is that the information should be machine readable, either by search engines and the like, or by anyone who wants to use our content and must take licencing into consideration.

It seems that many people simply include rel="licence" in the page content, i.e. 

    <a rel="license" href="http://creativecommons.org/licenses/by/3.0/">Creative Commons By 3.0</a>. 

Although it seems rel="licence" is no longer recommended by W3C in RDFa: https://www.w3.org/TR/xhtml-rdfa-primer/#links-with-flavor-1. They now recommend:

    <a property="http://creativecommons.org/ns#license" href="http://creativecommons.org/licenses/by/3.0/">
     a Creative Commons License</a>

Perhaps that's all we need, but it seems to me that the head is appropriate; it's simple, clear, and predictable. Having said which, perhaps we would get into trouble by confusing different licences for content on the page; the design, header, and so on are under a different licence than the text. But I doubt if this is a serious issue.

How to go about it? First check the copyright page on SC: `https://suttacentral.net/copyright`

This gives us some detail. 

1. The original texts and all data (including division pages and the like) are Public Domain. They are intrinsically exempt from copyright.
2. Original materials produced by SC are CC0, that is, they are potentially subject to copyright, but the creators have dedicated the work to the Public Domain. This includes this like the About page, Introduction, and so on. But this is trivial. What matters is the translations that we produce via Pootle. I have already developed a policy that ensures that all material created on Pootle is CC0: `https://pootle.suttacentral.net/pages/copyright/` So when we develop our scripts to export from Pootle, we should ensure that all such material, whatever format we export it to, is accompanied by a CC0 licence. (This, by the way, is more generally true of stuff exported by Pootle, we should also preserve other metadata such as translators, dates, and so on.)
3. What remains is the bulk of the  translations on SC. These are covered by a variety of formal and informal licences. Generally speaking, they are permitted for non-commercial use. But it is hard to make generalizations. Some may be labelled no adaptation, and so on. In many cases the licences are ad hoc. In other cases translators have agreed for SC to use them, but not beyond that. What I would suggest is that we parse the sidebar meta content and look for a Creative Common licence. If one is mentioned there, we extract that and insert that specific licence in the HTML metadata. If not, we simply say "other" or something, and refer users to the file info.

So I think we should see which of our texts are appropriately marked, ensure that they are up to date with the latest standards, and that we are consistent so far as possible. It's not a matter of major priority, just keeping the house in order.

If updating links, note also that Creative Commons has switched to https.

**Comment by aminah-sc** [comment]
Just linking this into https://github.com/suttacentral/suttacentral/issues/1028
